app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -47,6 +52,9 @@ public View display(final Context context, int position, View v) {
         TextView appName = (TextView) v.findViewById(R.id.item_app_name);
         appName.setText(enrichText(appPojo.displayName));
 
+        TextView tagsView = (TextView) v.findViewById(R.id.item_app_tag);
+        tagsView.setText(enrichText(appPojo.displayTags));
```

Comment:
Ah I get your point. I was thinking of them as shortcuts, because I have been using this branch for a couple days and everytime I type chrome I see "browser bla bla bla" and it's weird to me.

I don't care that much however, just in case anyone has a second opinion?